pvlib/atmosphere.py

```diff
@@ -683,3 +683,153 @@ def angstrom_alpha(aod1, lambda1, aod2, lambda2):
     pvlib.atmosphere.angstrom_aod_at_lambda
     """
     return - np.log(aod1 / aod2) / np.log(lambda1 / lambda2)
+
+
+def AM_AOD_PW_spectral_correction(airmass_absolute, aod500, pw,
```

Comment:
Would it be preferable to us the main author's name in the function name?

Reply:
Ok, taking into account the authors' initials and in order to make the pronunciation of the name easy (as far as possible), the name I propose is: jaemfag_spectral_corrections